src/map/unit.c

```diff
@@ -157,305 +206,295 @@ static int unit_walktoxy_sub(struct block_list *bl)
 	}
 	clif->move(ud);
 
-	if(ud->walkpath.path_pos>=ud->walkpath.path_len)
-		i = -1;
-	else if(ud->walkpath.path[ud->walkpath.path_pos]&1)
-		i = status->get_speed(bl)*MOVE_DIAGONAL_COST/MOVE_COST;
+	int timer_delay;
+	if (ud->walkpath.path_pos >= ud->walkpath.path_len)
+		timer_delay = -1;
+	else if ((ud->walkpath.path[ud->walkpath.path_pos] & 1) != 0)
+		timer_delay = status->get_speed(bl) * MOVE_DIAGONAL_COST / MOVE_COST;
 	else
-		i = status->get_speed(bl);
-	if( i > 0)
-		ud->walktimer = timer->add(timer->gettick()+i,unit->walktoxy_timer,bl->id,i);
-	return 1;
+		timer_delay = status->get_speed(bl);
+	if (timer_delay > 0)
```

Comment:
Please prepend a blank line, to clearly separate it from the if-else if-else block above.

Reply:
Done.